roles/katello_devel/meta/main.yml

```diff
@@ -20,4 +20,7 @@ dependencies:
       - "{{ '--katello-devel-github-username=' + katello_devel_github_username if katello_devel_github_username is defined else '' }}"
       - "--katello-devel-extra-plugins theforeman/foreman_rh_cloud"
       - "--katello-devel-extra-plugins theforeman/foreman_virt_who_configure"
```

Comment:
Probably also a question for @chris1984 , because you added this in https://github.com/theforeman/forklift/pull/1783

I would rather have it in `centos9-katello-devel` sample definition (see diff), so one can opt-out :grin: .

Reply:
I am fine with that, we just added it since we were doing a lot of work with virt-who at the time and also now doing work with rh_cloud. Feel free to move it out @m-bucher